services/identity/service.go

```diff
@@ -39,14 +43,14 @@ func init() {
 // Service handles identities
 type Service struct {
 	*sda.ServiceProcessor
-	identities      map[string]*storage
+	Identities      map[string]*Storage
 	identitiesMutex sync.Mutex
```

Comment:
`Identities` is public but you have an `identitiesMutex` private. I assume here the mutex is for controlling the access to `Identities` ? If `Identities` is public, then any user can access it without the mutex. It would make more sense to either enforce the use of the mutex by getter/setter or just drop the mutex if it's not necessary .